src/plot_settings.py

```diff
@@ -0,0 +1,177 @@
+"""
+plot_settings.py
+================
+
+usage:
```

Comment:
Nice! I like  the section about how to use!

Reply:
thanks :)